### ISSUE TITLE
Documentation example for `fork` seems incorrect

### ISSUE THREAD

**Issue by Toltar** [rayepps/radash#35]
Just found that the [documentation for `fork`](https://radash-docs.vercel.app/docs/array-fork) seems incorrect.

![image](https://user-images.githubusercontent.com/6003536/187313287-c1673059-4808-4566-b85d-fe283414030e.png)

I think the example code should be:

```
import { fork } from 'radash'

const gods = [{
  name: 'Ra',
  power: 100
}, {
  name: 'Zeus',
  power: 98
}, {
  name: 'Loki',
  power: 72
}, {
  name: 'Vishnu',
  power: 100
}]

const [finalGods, lesserGods] = fork(gods, f => f.power > 90) // [[ra, vishnu, zues], [loki]]
```

Since we are forking an array for all gods with a power level over 90.

**Comment by rayepps** [comment]
That is correct! Probably some copy and paste gone bad. Thanks for bringing this up. I'm gonna transfer this issue over to the docs repo.
